src/providers/index.js

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import {Sidebar} from '../contexts/sidebarContext'
+import {FilterProvider} from '../contexts/filterContext'
+
+const Providers=({children})=>{
+    return <Router>
+    <FilterProvider>
+        <Sidebar>{children}</Sidebar>
+    </FilterProvider>
+    </Router>
```

Comment:
Use something like prettier to auto-format this code.

Reply:
ok